src/app/store/RelatedProducts/RelatedProducts.reducer.js

```diff
@@ -32,6 +33,21 @@ const RelatedProductsReducer = (state = initialState, action) => {
             });
         }
 
+        const { items } = products;
```

Comment:
please move this but to action destruction and use already destructed property in line 28